src/components/ChatEntry.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
 
 const ChatEntry = (props) => {
+  const buttonContent = props.liked ? '❤️' : '🤍';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local'}>
```

Comment:
👍 Nice calculation of the local or remote class based on the sender. We could take this a step farther and tell the ChatEntry through its props the name of the local user (calculated somehow from the message data, say, maybe assume the first message is local) rather than hard coding a name in particular.